src/util/DefaultValueProcessor.ts

```diff
@@ -0,0 +1,35 @@
+import * as _ from "lodash";
+
+export const addValues = (testCase: any): any => {
+  //create a clone of testCase
+  const resultJson: any = _.clone(testCase);
+  //.map to return an array of just
+  const nonCoverage: Array<any> = resultJson.entry.filter((entry) => {
+    if (entry.resource?.resourceType !== "Coverage") {
+      return true;
+    }
+  });
+
+  let coverage: Array<any> = [];
+  const foundCoverage: Array<any> = testCase.entry?.filter((entry) => {
+    if (entry.resource?.resourceType === "Coverage") {
+      entry.resource.status = "active";
+      return entry;
+    }
+  });
+
+  if (foundCoverage) {
+    coverage = [...coverage, ...foundCoverage];
```

Comment:
`coverage` is always an empty array at this point, so it might be more logical to use `coverage.push(...foundCoverage)`.  This also allows `coverage` to be a `const`.